packages/prime-core/package.json

```diff
@@ -26,7 +26,7 @@
     "start": "node lib/index.js",
     "prepublishOnly": "yarn compile",
     "compile": "rm -rf ./lib && tsc -p .",
-    "dev": "DEBUG=prime* ts-node src",
+    "dev": "DEBUG=prime* node --inspect -r ts-node/register src",
```

Comment:
oops, forgot to remove this... thoughts on keeping it?

Reply:
ahah, great: https://github.com/birkir/prime/pull/154/files#diff-da0f3a2ff5bcf5e505b79f998eed0084L30

Reply:
Sure, if you want to add inspect to the dev:debug that sounds good!